python/simple/v1/plugins.py

```diff
@@ -0,0 +1,37 @@
+from threading import RLock
+from typing import Optional
+
+from amplitude import EventPlugin, DestinationPlugin, PluginType, BaseEvent, \
+    IdentifyEvent
+from analytics import Client as SegmentClient
+
+
+class MyEventIDPlugin(EventPlugin):
```

Comment:
Since these plugins can be used in all examples maybe we should move to a shared directory for easier maintaince e.g. `python/plugins/v1` ?

Reply:
I moved the plugins.py to python/plugins/v1 and tried append sys.path + importlib to import this to app.py and views.py. It's looks very ugly in apps.py and Django import external lib is harder. So I write in readme to copy the plugins module before running example apps. Is this OK?

Reply:
Ok, that's too bad. Let's focus on making this easy for the users to use and move the plugins back into the individual examples.